src/Container/ObjectRepositoryRoleProviderFactory.php

```diff
@@ -33,8 +34,11 @@
  */
 final class ObjectRepositoryRoleProviderFactory
```

Comment:
maybe remove this factory?
@prolic @basz

Reply:
for me, it's not that hard to create a factory so I would say remove. And maybe if they have config somewhere else or something they anyway have to create it. But for most people, this is enough so don't know remove or no.